ts/Core/Series/Series.ts

```diff
@@ -816,6 +816,19 @@ class Series {
             ),
             userPlotOptionsType = userPlotOptions[this.type] || {};
 
+        // When plot options have multiple data labels, the default data label
+        // plot options are not merged (#21928).
```

Comment:
This comment describes a situation that is no longer true. Instead, describe what the code actually does, and add a reference to the issue.
```suggestion
        // Merge in multiple data label options from the plot option (#21928).
```